core/src/services/huggingface/docs.md

```diff
@@ -0,0 +1,63 @@
+This service will visit the [HuggingFace API](https://huggingface.co/docs/huggingface_hub/package_reference/hf_api) to access the HuggingFace File System.
+Currently, we only support the `model` and `dataset` types of repositories, and operations are limited to reading and listing/stating.
+
+HuggingFace doesn't host official HTTP API docs. Detailed HTTP request API information can be found on the [HuggingFace Hub](https://github.com/huggingface/huggingface_hub).
```

Comment:
Technically, `huggingface_hub` is the name of the Python library to interact with the Hugging Face Hub (the online platform). So https://huggingface.co/docs/huggingface_hub/package_reference/hf_api is the package reference of the Python library (which is a good starting point for retro engineering). There is some documentation about the API itself (see [here](https://huggingface.co/docs/hub/api)) but it's not complete yet.